### PR TITLE
[Blog] Tables are Hard, Part 3: Streaming Data

### PR DESCRIPTION
**Pull request by NickLanam**
Signed-off-by: Nick Lanam <nlanam@pixielabs.ai>

### CODE REVIEW

File: content/blog/tables-are-hard-2/index.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ This tutorial is aimed at relative beginners to web development, and will keep t
 
 [Final Code](https://github.com/pixie-io/pixie-demos/tree/main/react-table) | [Live Demo](https://pixie-io.github.io/pixie-demos/react-table)
 
+_This article is part of a series_:
```
Comment (htroisi) [inline content/blog/tables-are-hard-2/index.md:22]: I'd move this below the video on the next line.

File: content/blog/tables-are-hard-2/index.md [hunk 2]
```diff
@@ -19,6 +19,11 @@ This tutorial is aimed at relative beginners to web development, and will keep t
...
+- [Part 1: History](https://blog.px.dev/tables-are-hard-3)
```
Comment (htroisi) [inline content/blog/tables-are-hard-2/index.md]:
```suggestion
- [Part 1: History](/tables-are-hard-1)
```

File: content/blog/tables-are-hard-1/index.md [hunk 3]
```diff
@@ -15,6 +15,11 @@ Yet modern software packs a surprising amount of functionality into them, accumu
 In this post, we'll take a high altitude pass over that history.
 In later installments, we'll dive into how today's software handles the complex requirements of modern tables.
 
+_This article is part of a series_:
+- **Part 1: History**
+- [Part 2: Building a Simple Data Table in React](https://blog.px.dev/tables-are-hard-2)
```
Comment (htroisi) [inline content/blog/tables-are-hard-1/index.md:20]:
```suggestion
- [Part 2: Building a Simple Data Table in React](/tables-are-hard-2)
```

File: content/blog/tables-are-hard-1/index.md [hunk 4]
```diff
@@ -15,6 +15,11 @@ Yet modern software packs a surprising amount of functionality into them, accumu
...
+- [Part 3: Streaming Data](https://blog.px.dev/tables-are-hard-3)
```
Comment (htroisi) [inline content/blog/tables-are-hard-1/index.md:21]:
```suggestion
- [Part 3: Streaming Data](/tables-are-hard-3)
```

File: content/blog/tables-are-hard-1/index.md (hunk 3 above)
Comment (htroisi) [reply content/blog/tables-are-hard-1/index.md:20]: Check that this works, but I think you want relative links for these.

File: content/blog/tables-are-hard-2/index.md [hunk 5]
```diff
@@ -19,6 +19,11 @@ This tutorial is aimed at relative beginners to web development, and will keep t
...
+- **Part 2: Building a Simple Data Table in React**
+- [Part 3: Streaming Data](https://blog.px.dev/tables-are-hard-3)
```
Comment (htroisi) [inline content/blog/tables-are-hard-2/index.md]:
```suggestion
- [Part 3: Streaming Data](/tables-are-hard-3)
```

File: content/blog/tables-are-hard-3/index.md [hunk 6]
```diff
@@ -0,0 +1,532 @@
...
+date: 2022-04-06T07:00:00.000+00:00
+featured_image: tables-are-hard-3-hero.png
+categories: ['Pixie Team Blogs']
+authors: ['Nick Lanam']
+emails: ['nlanam@pixielabs.ai']
+---
+
+Let's build a tailing log viewer in React. Or at least, a facsimile of one.
+
+In our [previous post](https://blog.px.dev/tables-are-hard-2/), we made a table with sorting, filtering, and column controls.
```
Comment (htroisi) [inline content/blog/tables-are-hard-3/index.md:13]:
```suggestion
In our [previous post](/tables-are-hard-2/), we made a table with sorting, filtering, and column controls.
```

File: content/blog/tables-are-hard-2/index.md (hunk 1 above)
Comment (NickLanam) [reply content/blog/tables-are-hard-2/index.md]: Good catch, done so it's the same between both articles.

File: content/blog/tables-are-hard-3/index.md [hunk 7]
```diff
@@ -0,0 +1,532 @@
...
+    <source src="./tables-are-hard-3-full-demo.mp4" type="video/mp4" />
+  </video>
+  <figcaption>
+    A quick video demonstration of what we'll build.
+  </figcaption>
+</figure>
+:::
+
+_This article is part of a series_:
+- [Part 1: History](https://blog.px.dev/tables-are-hard-1)
```
Comment (htroisi) [inline content/blog/tables-are-hard-3/index.md:33]:
```suggestion
- [Part 1: History](/tables-are-hard-1)
```

File: content/blog/tables-are-hard-3/index.md [hunk 8]
```diff
@@ -0,0 +1,532 @@
...
+- [Part 2: Building a Simple Data Table in React](https://blog.px.dev/tables-are-hard-2)
```
Comment (htroisi) [inline content/blog/tables-are-hard-3/index.md:34]:
```suggestion
- [Part 2: Building a Simple Data Table in React](/tables-are-hard-2)
```

**Review by htroisi — COMMENTED** [review]
Could you add a line at the bottom of Pt 1 and Pt 2 that links to the next part in the series? Something like "Check out Part 2 to see how..."

**Review by htroisi — APPROVED** [review]

**Pull request merged**